keps/sig-auth/267-kubelet-server-certificate-bootstrap-rotation/kep.yaml

```diff
@@ -0,0 +1,28 @@
+title: Kubelet Server TLS Certificate Rotation
+kep-number: 267
+authors:
+  - "@mikedanese"
+  - "@liggitt"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
+  - sig-node
```

Comment:
Is SIG Security involved at all? Might they want to be?

Reply:
added